src/tests/user.test.ts

```diff
@@ -1,44 +1,166 @@
+});
+
+// used when querying for a user
+const userResponse = Object.freeze({
+    email:       'coolman@example.com',
+    displayName: 'cool man',
+    id:          'aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee',
+});
+
+// used for authenticating a user that doesn't exist
+const undefinedUser = Object.freeze({
+    email:       'does_not_exist@example.com',
+    password:    'passWord1',
+    displayName: 'user does not exist',
+    id:          'AAAAAAAA-bbbb-cccc-dddd-eeeeeeeeeeee'
+});
+
+describe('Create User', () => {
+    test('Creating a user should return the new user', async () => {
+        (userService.createUser as jest.Mock).mockReturnValue(() => Promise.resolve(userResponse));
+        const newUser = await _createUserController(sequelizeMock())(user.email, user.password, user.displayName);
+        expect(newUser).toEqual(userResponse);
     });
 
-    test('If the user exists, it should be returned', async () => {
-        const userData = {
-            password:     'Password!',
-            email:        'email@yahoo.com',
-            display_name: 'display'
-        };
+    test('Create user with bad data should throw an error', async () => {
+        (userService.createUser as jest.Mock).mockReturnValue(() => { throw new Error(); });
+        const newUser = await _createUserController(sequelizeMock())(undefined as any, undefined as any, undefined as any);
+        expect(newUser).toBeInstanceOf(Error);
+    });
 
-        (userService.getUserByID as jest.Mock).mockReturnValue(() => Promise.resolve(userData));
+    test('Create user with poorly formed email should throw a validation error', async () => {
+        (userService.createUser as jest.Mock).mockReturnValue(() => { throw new ValidationError('Invalid email', []); });
+        const newUser = await _createUserController(sequelizeMock())(user.badEmail, user.password, user.displayName);
+        expect(newUser).toBeInstanceOf(ValidationError);
+    });
 
-        const response = await _getUserByIDController(sequelizeMock)('8c2d50d1-9b1f-483c-b267-cecb929ffb97');
+    test('Create user with short display name should throw a validation error', async () => {
+        (userService.createUser as jest.Mock).mockReturnValue(() => { throw new ValidationError('Invalid display name', []); });
+        const newUser = await _createUserController(sequelizeMock())(user.badEmail, user.password, '');
+        expect(newUser).toBeInstanceOf(ValidationError);
+    });
 
-        expect(response).toEqual(userData);
+    test('Create user with long display name should throw a validation error', async () => {
+        (userService.createUser as jest.Mock).mockReturnValue(() => { throw new ValidationError('Invalid display name', []); });
+        const newUser = await _createUserController(sequelizeMock())(user.badEmail, user.password, '>30 chars aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa');
+        expect(newUser).toBeInstanceOf(ValidationError);
```

Comment:
These tests seem unnecessary. The invalid inputs are not the reason why the `ValidationErrors` are being thrown, but rather because you are telling it to. There should only be two tests for this method:
1. returning the correct data
2. catching and returning a thrown error

Reply:
The validation error happens within the service function, since the sequelize model itself has validators. I thought checking this mattered since the error was being thrown from the service function rather than the controller, but if all we're doing is checking the controller as if it was a black box, I can remove the test